brightray/brightray.gyp

```diff
@@ -199,7 +199,6 @@
                 'libraries': [
                   # Following libraries are always linked statically.
                   '<(libchromiumcontent_dir)/libhttp_server.a',
-                  '<(libchromiumcontent_dir)/libdesktop_capture.a',
```

Comment:
@nitsakh 
Then we shouldn't build it in libcc
https://github.com/electron/libchromiumcontent/blob/chromium-upgrade/66/chromiumcontent/BUILD.gn#L98

Reply:
Right, I am making that change.